ReadMe.md

```diff
@@ -77,6 +78,24 @@ sudo apt-get install gcc-8 g++-8
 sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8
 ```
 
+### macOS prerequisites
+Make sure you have all the dependencies installed (assuming brew):
+```bash
+brew install coreutils
+brew install qemu
+brew cask install osxfuse
+Toolchain/BuildFuseExt2.sh
+```
+
+Notes: 
+- Assuming xcode and xcode-tools are installed (git is required by some scripts)
+- coreutils is needed for building gcc
+- qemu is needed to run the actual OS. As an alternative you can Build it using the BuildQemu.sh script
+- osxfuse and BuildFuseExt2.sh are needed if you want to build the root filesystem disk image natively on macOS. This allows mounting an EXT2 fs and also installs commands like mke2fs that are not available on stock macos. 
+- If you install some commercial EXT2 macOS fs handler instead of osxfuse and fuse-ext2, you will need to install e2fsprogrs using brew to obtain mke2fs.
```

Comment:
Please format script names and such with `backticks` as it's done in other sections

Reply:
Ok, sure

Reply:
I have tried formatting it using `backticks`, please let me know if it looks more or less right now.